src/cmd/apply.go

```diff
@@ -245,7 +245,10 @@ func pushExtensions(destExt string, list ...string) {
 			extPath = v
 		} else {
 			extName = v
-			extPath, err = utils.GetExtensionPath(v)
+			if(!strings.Contains(extName,".")){
```

Comment:
I'd say to verify if it has `".js"` instead since a file name can have a . without it being the extension.

Reply:
@afonsojramos Done.